README.md

```diff
@@ -143,12 +144,14 @@ longer deemed necessary, then a project can also remove the directory.
 
 Conversion of a repository that uses `dep` to `go mod` can be done with:
 
+```shell
```

Comment:
Please remove the indention inside the block.

Reply:
done